spec/services/doc_auth_router_spec.rb

```diff
@@ -74,7 +74,7 @@ def reload_ab_test_initializer!
     context 'with a nil discriminator' do
       let(:discriminator) { nil }
 
-      it 'is the default vendor, and logs analytics events' do
+      it 'is the default vendor, and logs analytics events', allowed_extra_analytics: [:error] do
```

Comment:
Should this be allowed? 🤔

Reply:
I think no, this is what I was [referencing in the PR description](https://github.com/18F/identity-idp/pull/9946#:~:text=We%20have%20analytics%20methods%20that%20don%27t%20take%20params%20but%20do%20add%20extra%20stuff%20to%20the%20payload%20as%20new%20undocumented%20keys%20like%20so%3A), where the `idv_doc_auth_randomizer_defaulted` takes no params (so there's nothing to document) but adds extra info in the body of the method :/

Reply:
I decided to retry this in d4efd77f6a155d8933468e18c73fe8513a5ddb3d and removed it by changing the analytics event to use default keyword arguments

Reply:
Ah, I think I missed that the place where the parameter was passed was within `analytics_events.rb` itself.